src/mumble/OverlayConfig.cpp

```diff
@@ -570,5 +586,37 @@ void OverlayConfig::on_qpbSavePreset_clicked() {
 	qs.remove(QLatin1String("usewhitelist"));
 	qs.remove(QLatin1String("blacklist"));
 	qs.remove(QLatin1String("whitelist"));
+	qs.remove(QLatin1String("enablelauncherfilter"));
+	qs.remove(QLatin1String("launchers"));
 	qs.endGroup();
 }
+
+void OverlayConfig::on_qcbEnableLauncherFilter_stateChanged(int state) {
+	qlLauncherFilterDescription->setEnabled(state == Qt::Checked);
+	qlwLaunchers->setEnabled(state == Qt::Checked);
+	qpbAddLauncher->setEnabled(state == Qt::Checked);
+	qpbRemoveLauncher->setEnabled(state == Qt::Checked);
+}
+
+void OverlayConfig::on_qpbAddLauncher_clicked() {
+	QString file = QInputDialog::getText(this, tr("Add launcher"), tr("Add a launcher executable to the Overlay Launcher Filter..."));
```

Comment:
I did it mostly to be able to add launchers that aren't on your system, or launchers you don't know where live.

I know this is somewhat far-fetched.

But I like to have it free-form.

I guess it's also easier to guide users to whitelist a launcher by telling them to add "XYZLauncher.exe" rather than to "*sigh* [...] click on the C-drive, then go into Program Files, then... [...]".

I don't know.